src/main/java/net/snowflake/ingest/utils/ErrorCode.java

```diff
@@ -14,7 +14,7 @@ public enum ErrorCode {
   REGISTER_BLOB_FAILURE("0006"),
   OPEN_CHANNEL_FAILURE("0007"),
   BUILD_REQUEST_FAILURE("0008"),
-  CLIENT_CONFIGURE_FAILURE("0009"),
+  CONFIGURE_FAILURE("0009"),
```

Comment:
lets not reuse error codes between client configure and channel configure, these are two independent APIs.

Reply:
Created separate error codes.